packages/suite/src/hooks/suite/useEverstakePoolStats.ts

```diff
@@ -1,8 +1,15 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useMemo, useState } from 'react';
 import BigNumber from 'bignumber.js';
-import { BACKUP_ETH_APY } from 'src/constants/suite/ethStaking';
+import { BACKUP_ETH_APY, STAKE_SYMBOLS } from 'src/constants/suite/ethStaking';
+import { selectEnabledNetworks } from 'src/reducers/wallet/settingsReducer';
+import { useSelector } from './useSelector';
 
 export const useEverstakePoolStats = () => {
+    const enabledNetworks = useSelector(selectEnabledNetworks);
```

Comment:
When you added this selector it's now casing unit tests to fail. You can look at the validation job in CI.

To fix this you must mock the store since `enabledNetworks.some` are throwing undefined.
![EFD8BE56-5CB9-45EF-B040-0EB1FE02CCD7_4_5005_c](https://github.com/trezor/trezor-suite/assets/66002635/eb763a34-e4d0-43b2-94ee-da220ff39bc7)

Hint: Add `enabledNetworks: ['thol']` to line 60 in `useSendForm.test.ts` in the settings object and it's passing.

Reply:
done